### ISSUE TITLE
Adaptive card or other type of cards rendering issue

### ISSUE THREAD

**Issue by CalvinFengDatacom** [zixia/botbuilder-wechaty-connector#28]
Hi there,

I'm a bot developer has many experience of developing chat bot by MS bot F/W in Web Chat, Facebook, and Skype. I can see both Facebook and Skype channel are not support rendering adaptive card. So, I'm quite interesting about whether the WeChat channel can render the Adaptive card or any other type of cards?

I'd like to have a discussion about this topic in this channel :-)

Happy to see any responses

**Comment by huan** [comment]
Hi @CalvinFengDatacom,

It's great to hear from bot developers who have many experiences of developing chatbots like you.

About your question: does wechaty-adapter WeChat channel can render the cards?

The answer is: sorry, we can't.

That's because this adapter is for an individual account, and individual account in WeChat cannot send cards.

However, if you use the Official Account adapter, I believe there's possible to let you have cards support in WeChat, because of the Official Account support cards.

Are you working with WeChat? What kind of bot are you building?

Please let me know if you have any progress, and you can add my Wechat at 918999.

Have a nice day!

**Comment by CalvinFengDatacom** [comment]
@huan Thanks for your response, we are mainly creating ChatBot on the Web channel and Facebook channel. But from our experience, we can see the adaptive card does not rendering very well in Facebook channel. So I'm curious whether WeChat supports it. Currently, we don't have any chance of creating a ChatBot on WeChat.

**Comment by huan** [comment]
As far as I know, the Wechat channel will no better than the Facebook channel.

Here has a simple guide for Wechat Official Account, you can have a look at it and it might be able to give you some insights: <https://walkthechat.com/wechat-official-account-simple-guide/>

**Comment by Techie921** [comment]
Hi @huan ,
Can we integrate this adapter with LUIS app?

Thanks
Sakru.

**Comment by huan** [comment]
@techie921 of course, you can integrate this adapter with LUIS app because it is a botbuilder channel adapter.

**Comment by Techie921** [comment]
Hi,

Have you done this before?

If so, please share code sample

Thanks,
Sakru

**Comment by Techie921** [comment]
Hi @huan ,

I have tried it but could not succeed in it. Could you please share the code if you have tried it.

Thanks,
Sakru Jarpula.

**Comment by huan** [comment]
@techie921 I want to share with you but I'm sorry that I have no code in my hand right now.

If you need help, please you feel free to post your full source code to this issue and I'd like to have a look on it for you.
